cobc/help.c

```diff
@@ -129,6 +130,8 @@ cobc_print_usage_common_options (void)
 	puts (_("  --list-system         display system routines"));
 	puts (_("  --save-temps[=<dir>]  save intermediate files\n"
 	        "                        * default: current directory"));
+	puts (_("  -MT <target>          set/add target file used in dependency list"));
+	puts (_("  -MF <file>            place dependency list into <file>"));
```

Comment:
this reminds me of #109 - I really hope "someone" can take this up, starting with the rebase...

Reply:
I notified the team about it.